src/karts/abstract_kart.cpp

```diff
@@ -79,6 +99,12 @@ void AbstractKart::reset()
 }   // reset
 
 // ----------------------------------------------------------------------------
+const AbstractCharacteristic* AbstractKart::getCharacteristic() const
+{
+    return &(*m_characteristic);
```

Comment:
Ups, sorry I misread your comment, I thought you meant I should return a unique_ptr... `get()` looks nice :smiley: